helix-core/src/main/java/org/apache/helix/controller/rebalancer/util/DelayedRebalanceUtil.java

```diff
@@ -142,7 +143,9 @@ private static long getInactiveTime(String instance, Set<String> liveInstances,
       if (clusterConfig.getDisabledInstances() != null && clusterConfig.getDisabledInstances()
           .containsKey(instance)) {
         // Update batch disable time
-        long batchDisableTime = Long.parseLong(clusterConfig.getDisabledInstances().get(instance));
+        long batchDisableTime = Long.parseLong(
```

Comment:
Could parseLong() be called on `null`?

Reply:
We always add the Timestamp when batch enabling instance. I don't think it could be null unless ZNode is manually updated.